hal/hal_remote/CMakeLists.txt

```
@@ -25,7 +25,7 @@ set(HAL_SOURCE
   ${CMAKE_CURRENT_SOURCE_DIR}/source/hal_server.cpp
 )
 
-if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux" OR CMAKE_SYSTEM_NAME STREQUAL "Darwin")
```

I'd rather we didn't add macOS-specific logic until we're ready to actually support that, but you can achieve essentially the same effect with
```suggestion
if(NOT CMAKE_SYSTEM_NAME STREQUAL "Windows")
```
which I'd be fine with, if that works.

---

we already have Darwin stuff in OCK - but I'll got with not "windows"